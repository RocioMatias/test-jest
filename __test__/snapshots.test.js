import { getCharacter } from '../snapshots';
import rick from '../rick.json';

describe('Es hora de las instantaneas', () => {
    test('Snapshot', () => {
        expect(getCharacter(rick)).toMatchSnapshot();
    });
    test('siempre fallara la instantanea', () => {
        const user = {
            createAt: new Date(),
            id: Math.floor(Math.random() * 20),
        }
        expect(user).toMatchSnapshot();
    });
    test('permitimos una expecion', () => {
        const user = {
            id: Math.floor(Math.random() * 20),
            name: 'Rocio Matias'
        }
        expect(user).toMatchSnapshot({
            id: expect.any(Number)
        });
    });
});