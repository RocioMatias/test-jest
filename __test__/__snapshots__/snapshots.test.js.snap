// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Es hora de las instantaneas Snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanches",
  "species": "Human",
  "status": "Alive",
}
`;

exports[`Es hora de las instantaneas permitimos una expecion 1`] = `
Object {
  "id": Any<Number>,
  "name": "Rocio Matias",
}
`;

exports[`Es hora de las instantaneas siempre fallara la instantanea 1`] = `
Object {
  "createAt": 2020-07-07T03:51:42.647Z,
  "id": 18,
}
`;
